internal/generator/vector/output/otlp/transform.go

```diff
@@ -110,7 +110,13 @@ if exists(kv.type) {
   r.attributes = push(r.attributes, {"key": "auditd.type", "value": {"stringValue": kv.type }})
 }
 if exists(kv.msg) {
-  trimmed = slice!(kv.msg, find!(kv.msg, "(") + 1, -2)
+  msg_str = ""
+  if is_array(kv.msg) {
+    msg_str = kv.msg[0] 
```

Comment:
How do we know that only the first message is relevant?

Reply:
This is standard `auditd` log format: `type=TYPE msg=audit():`

https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/7/html/security_guide/sec-understanding_audit_log_files#sec-Understanding_Audit_Log_Files